sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample_RecognizeHealthcareEntities.md

```diff
@@ -42,12 +46,17 @@ To recognize healthcare entities in multiple documents, call `StartAnalyzeHealth
     AnalyzeHealthcareEntitiesOperation healthOperation = await client.StartAnalyzeHealthcareEntitiesAsync(batchInput, "en", options);
 
     await healthOperation.WaitForCompletionAsync();
+```
+To view the operation details:
```

Comment:
Idk if an extra space will help to distinguish this a little more.
Also, it might be good to change the wording to something like 
```
The healthcare operation contains general information about the status of the operation. It can be requested while the operation is running or when it completed. For example:
```